README.md

```diff
@@ -1,3 +1,43 @@
-# flatpak-platform
+<div align="center">
+  <h1 align="center"><center>flatpak-platform</center></h1>
+  <h3 align="center"><center>An elementary OS Flatpak platform</center></h3>
+  <br>
+  <br>
+</div>
 
-A platform for Flatpak from elementary OS.
+<p align="center">
+  <img src="https://github.com/elementary/flatpak-platform/workflows/Daily/badge.svg" alt="Daily">
+</p>
+
+---
+
+## Building
+
+If you want to build this flatpak platform locally, you will need the flathub flatpak repository installed. You can
+follow instructions for that [here](https://flatpak.org/setup/).
+
+Next you will need:
+
+- A lot of space (~ 15 Gb)
+- `flatpak-builder`
+- `git`
+
+And finally, you can run this command to build and install:
+
+```sh
+flatpak-builder --force-clean --install-deps-from=flathub --ccache --repo=elementary --install builddir ./io.elementary.Sdk.json
+```
```

Comment:
Meson seemed really over the top for running this one command. It also made it a bit harder to get working in CI, so I figured we can just drop it for a simpler solution.